include/pnet_api.h

```diff
@@ -1171,23 +1171,19 @@ typedef struct pnet_ethaddr
    (PNET_STATION_NAME_MAX_SIZE + PNET_PORT_NAME_MAX_SIZE)
 
 /**
- * Network interface
+ * Network interface name
  */
-typedef struct pnet_netif
+typedef struct pnet_netif_name
 {
-   char if_name[PNET_INTERFACE_NAME_MAX_SIZE]; /**< Terminated string */
-   pnet_ethaddr_t eth_addr;                    /**< Interface MAC address */
-} pnet_netif_t;
+   char name[PNET_INTERFACE_NAME_MAX_SIZE]; /**< Terminated string */
+} pnet_netif_name_t;
 
 /**
  * Physical Port Configuration
  */
 typedef struct pnet_port_cfg
 {
-   pnet_netif_t phy_port;
-   char port_name[PNET_PORT_NAME_MAX_SIZE]; /**< Terminated string */
-   uint16_t rtclass_2_status;
-   uint16_t rtclass_3_status;
+   pnet_netif_name_t port_netif;
```

Comment:
```suggestion
   const char * network_interface_name;
```
(Assuming that pnet_cfg_t is no longer stored in pnet_t.)

Reply:
Remove pnet_cfg_t from pnet_t is out of the scope for this task. I will use name "netif_name".

Reply:
Beware that the copy of pnet_cfg_t in pnet_t will be a shallow copy, unless the strings are also copied.